### PR TITLE
README with instructions on using AsciiDoc and Markdown

### PR DESCRIPTION
**Pull request by Ben-Graham**
Guide includes instructions on how to make titles, subtitles, ordered lists, unordered lists, and source code blocks in AsciiDoc and Markdown files.

### CODE REVIEW

**Comment by enochtsang** [comment]
Can you remove the markdown, there should be one standard. By giving markdown instructions we enable future people to stray away from the standard.

**Comment by bill-luu** [comment]
But since markdown is the primary style on git, wouldn't it be useful to have these instructions? We could specify that astyle is the standard that the team uses, and that markdown is for git comments.

**Comment by enochtsang** [comment]
No, even git comments can use adoc format, so there's no real good reason for them to learn it, it just bloats our documentation.

File: Development-Guide/AsciiDoc-Guide/README.adoc [hunk 1]
```diff
@@ -0,0 +1,57 @@
+= AsciiDoc
+
+== Headers
```
Comment (enochtsang) [inline Development-Guide/AsciiDoc-Guide/README.adoc]: Best practice to leave a line of space above and below a header.

File: Development-Guide/AsciiDoc-Guide/README.adoc [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+When writing an AsciiDoc a large header can be made by writing
+
+....
+= A Desciptive Title
+....
+
+Similarly a sub header can be created by writing
```
Comment (enochtsang) [inline Development-Guide/AsciiDoc-Guide/README.adoc]: Explain instead that each equal sign gives it a proceeding header level. 

```
= Title

== h2

=== h3
...
```

File: Development-Guide/AsciiDoc-Guide/README.adoc [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+....
+== A Descriptive Subtitle
+....
+
+== Lists
+
+An unordered list, ie. a list of bullet points can be made with
```
Comment (enochtsang) [inline Development-Guide/AsciiDoc-Guide/README.adoc]: ... can be made with asterisks.

File: Development-Guide/AsciiDoc-Guide/README.adoc [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+[source,adoc]
+* First Item
+* Second Item
+
+A numbered list can be made as
```
Comment (enochtsang) [inline Development-Guide/AsciiDoc-Guide/README.adoc]: is done with periods.

File: Development-Guide/AsciiDoc-Guide/README.adoc [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+== Source Code
+
+To write source code in an AsciiDoc, write "[source,a language]"
+Lines after the square brackets will be displayed as a block of code.
+An empty line will end the block of code.
+For example:
+
+....
+[source, c]
```
Comment (enochtsang) [inline Development-Guide/AsciiDoc-Guide/README.adoc]: this goes above the `....` and normally people use `----`

File: Development-Guide/AsciiDoc-Guide/README.adoc [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+Some code
+Some more code
+....
+
+== Formatting
+
+In AsciiDocs it is best practice for each new sentence to be started on a new line in your text editor.
```
Comment (enochtsang) [inline Development-Guide/AsciiDoc-Guide/README.adoc]: AsciiDocs isn't really a term.

`When writing in asciidoc, each new sentence...`

File: Development-Guide/AsciiDoc-Guide/README.adoc (hunk 5 above)
Comment (Ben-Graham) [inline Development-Guide/AsciiDoc-Guide/README.adoc]: the [source, c] is part of the example for showing how source code is written.

**Pull request closed without merging**